examples/elasticache_memcached.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: service-operator.aws/v1alpha1
+kind: ElastiCache
+metadata:
+  name: memcached13
+spec:
```

Comment:
This is great could you do me one favor and make it so that the keys are `lowerCamel` style?

Reply:
No problem! Ok updated keys to use lowerCamel here and in model.